client/my-sites/plans/current-plan/thank-you-header/index.js

```diff
@@ -0,0 +1,90 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { get, isFinite, noop } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import getCurrentPlanPurchaseId from 'state/selectors/get-current-plan-purchase-id';
+import ProgressBar from 'components/progress-bar';
+import QuerySitePurchases from 'components/data/query-site-purchases';
+import { getByPurchaseId } from 'state/purchases/selectors';
+import { getPlan } from 'lib/plans';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSitePlanSlug } from 'state/sites/selectors';
+
+export class PlanSetupHeader extends Component {
```

Comment:
I always get confused from cases where component name is very different from the file name it's into.

Reply:
Thanks, I missed cleaning this up 👍
db66a46eb877503ec8cb974121b1b5dc21cbbcd1